docs/en/02_Developer_Guides/09_Security/04_Secure_Coding.md

```diff
@@ -784,6 +788,12 @@ SilverStripe\Core\Injector\Injector:
       TokenCookieSecure: true
 ```
 
+[info]
+There is not currently an easy way to pass a `securesite` attribute value for setting this cookie - but you can set the
```

Comment:
Remove section

Reply:
I'll remove the bit that talks about adding an extension - but I think it's worth pointing at the `Cookie.default_samesite` config here.